tests/e2e-tests/src/deployment-api.test.ts

```diff
@@ -14,20 +19,21 @@ describe('DeploymentApi', () => {
   let initialState: AiDeploymentList | undefined;
 
   beforeAll(async () => {
-    const queryResponse = await DeploymentApi.deploymentQuery(
-      {},
-      { 'AI-Resource-Group': resourceGroup }
-    ).execute();
+    await cleanupDeployments();
+    const queryResponse = await getDeployments(resourceGroup);
```

Comment:
[q] When we clean up all deployments initially, does this return anything?

Reply:
Yes it returns the count. In the event that cleaning deployment fails initially, then this would still be useful for that particular test flow since we would be matching the initial and final state at the end.